postreise/analyze/demand.py

```diff
@@ -0,0 +1,44 @@
+from powersimdata.network.usa_tamu.usa_tamu_model import area_to_loadzone
+
+from postreise.analyze.generation.summarize import (
+    get_generation_time_series_by_resources,
+)
+
+
+def get_demand_time_series(scenario, area, area_type=None):
+    """Get time series demand in certain area of a scenario
+
+    :param powersimdata.scenario.scenario.Scenario scenario: scenario instance
+    :param str area: one of *loadzone*, *state*, *state abbreviation*,
+        *interconnect*, *'all'*
+    :param str area_type: one of *'loadzone'*, *'state'*, *'state_abbr'*,
+        *'interconnect'*
+    :return: (*pandas.Series*) -- time series of total demand, index: time stamps,
+        column demand values
```

Comment:
I think there should be a colon `:` after column

Reply:
Done.